hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1355,15 +1360,15 @@ public OzoneFileStatus getFileStatus(OmKeyArgs args) throws IOException {
     String bucketName = args.getBucketName();
     String keyName = args.getKeyName();
 
-    metadataManager.getLock().acquireBucketLock(volumeName, bucketName);
     try {
+      metadataManager.getLock().acquireBucketLock(volumeName, bucketName);
```

Comment:
why do we move the acquireBucketLock inside the try block? The original pattern seems good to me.